### PR TITLE
LineSymbolizer GraphicFill and GraphicStroke

### PR DESCRIPTION
**Pull request by jansule**
**Important:** Please wait with merging this PR until

- [x] [this PR from geostyler-style](https://github.com/terrestris/geostyler-style/pull/46)
was merged.

Now also parsing GraphicFill and GraphicStroke of a LineSymbolizer. Both can either be a wellKnownSymbol or an externalGraphic.

Updated tests and testfiles accordingly.

### CODE REVIEW

**Comment by jansule** [comment]
@terrestris/devs open for review.

File: data/slds/line_graphicFill.sld [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                </Graphic>
+              </GraphicFill>
+              <CssParameter name="stroke">#000000</CssParameter>
+              <CssParameter name="stroke-width">3</CssParameter>
+              <CssParameter name="stroke-dasharray">13 37</CssParameter>
+              <CssParameter name="stroke-linecap">round</CssParameter>
+              <CssParameter name="stroke-linejoin">mitre</CssParameter>
+            </Stroke>
+          </LineSymbolizer>
+       	</Rule>
```
Comment (dnlkoch) [inline data/slds/line_graphicFill.sld]: Very minor: Indentation.

File: src/SldStyleParser.ts [hunk 2]
```diff
@@ -439,7 +439,12 @@ class SldStyleParser implements StyleParser {
           break;
         case 'GraphicStroke':
           lineSymbolizer.graphicStroke = this.getPointSymbolizerFromSldSymbolizer(
-            _get(sldSymbolizer, 'Stroke[0].GraphicStroke[0]')
+            sldSymbolizer.Stroke[0].GraphicStroke[0]
+          );
+          break;
+        case 'GraphicFill':
+          lineSymbolizer.graphicFill = this.getPointSymbolizerFromSldSymbolizer(
+            sldSymbolizer.Stroke[0].GraphicFill[0]
```
Comment (dnlkoch) [inline src/SldStyleParser.ts]: I would prefer usage of `_get()` here (and above). Is there any reason you removed it?

Comment (dnlkoch) [reply src/SldStyleParser.ts]: Nope, your're right! My bad, haven't taken the context into account.

**Pull request merged**